src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -1444,36 +1591,45 @@ def fundamental_group_from_braid_mon(bm, degree=None, simplified=True, projectiv
 
     EXAMPLES::
 
+        sage: # optional - sirocco
```

Comment:
Same comment about what tests actually need sirocco.

Reply:
It can be changed but the tests are meaningful only when `sirocco` is installed.

Reply:
Same reasoning applies (as mentioned in the other comment).